src/viur/core/request.py

```diff
@@ -435,6 +435,10 @@ def _route(self, path: str) -> None:
                 f" of {conf['viur.maxPostParamsCount']} allowed arguments per request"
             )
 
+        param_filter = conf["viur.paramFilterFunction"]
+        if param_filter and not callable(param_filter):
+            raise ValueError(f"""{param_filter=} is not callable""")
```

Comment:
I think this check could be moved to a better place where it is not called on every request. But okay for `main`. On `develop`, we can validate it in the `config` object.

Reply:
>  On `develop`, we can validate it in the `config` object.

Can you either tell me how, or make the merge?

Reply:
Maybe with an `@property` like https://github.com/viur-framework/viur-core/blob/80b93191df5d6d8c426ac5181f8d6696c666c168/src/viur/core/config.py#L717-L747

In the longer term, we could also look to validate the values in setattr against the type hints.